modules/forms/client-react/FormError.ts

```diff
@@ -0,0 +1,41 @@
+import { isApolloError } from 'apollo-client';
+import { GraphQLError } from 'graphql';
+
+export const isFormError = (err: any) => err instanceof FormError;
+
+export class FormError {
+  public _errors: { [key: string]: any };
```

Comment:
I guess this field is not supposed to be `public`.

Reply:
Fixed